neoforge/build.gradle.kts

```diff
@@ -18,12 +18,14 @@ dependencies {
 
     val minecraftVersion: String by project
     val neoforgeVersion: String by project
-    val reiVersion: String by project
+    val xaerosWorldMapVersion: String by project
+    val xaerosMiniMapVersion: String by project
+    val byteCodecsVersion: String by project
 
     neoForge(group = "net.neoforged", name = "neoforge", version = neoforgeVersion)
 
-    forgeRuntimeLibrary("com.teamresourceful:bytecodecs:1.0.2")
+    forgeRuntimeLibrary(group = "com.teamresourceful", name = "bytecodecs", version = byteCodecsVersion)
```

Comment:
So the Neoforge version doesn't run. Sorry for not testing this I was using the Fabric version.

Might be related to me updating the resourcefullib version or may have not worked from the start.

It's having an issue where resourcefullib can't find a missing bytecodecs class. even though it should be included. Not quite sure how to fix yet.